src/main/scala/za/co/absa/abris/avro/parsing/utils/AvroSchemaUtils.scala

```diff
@@ -67,6 +68,34 @@ object AvroSchemaUtils {
     toAvroType(field.dataType, field.nullable, recordName, nameSpace)
   }
 
+  def toAvroSchema(
+    dataFrame: DataFrame,
+    columnNames: Seq[String]
+  ): Schema = toAvroSchema(dataFrame, columnNames, "topLevelRecord", "")
```

Comment:
Kind of a nitpick, but in the existing method
```
  def toAvroSchema(
    dataFrame: DataFrame,
    columnName: String,
    recordName: String = "topLevelRecord",
    nameSpace: String = ""
  ): Schema
```
you are specifying the default values for `recordName` and `nameSpace` directly in the parameters while here you use a separate method to specify the default values for the method below. I would stay consistent. Why not just use default values directly in the parameters?

Reply:
It's not possible to define multiple overloaded methods with default parameters in Scala.

https://stackoverflow.com/questions/4652095/why-does-the-scala-compiler-disallow-overloaded-methods-with-default-arguments

And I am keeping the old method as is for backwards compatibility...

Reply:
Ah sorry, I didn't know about this. My bad